app/helpers/application_helper.rb

```diff
@@ -28,6 +28,16 @@ def current_path_without_query_string
     request.original_fullpath.split("?", 2).first
   end
 
+  def has_saved_page_confirmation?
+    personalisation_param.present?
+  end
```

Comment:
One final comment: can you also put this behind the feature flag

Reply:
sure!